network.env.dist

```diff
@@ -67,65 +71,96 @@ ZONE_2_0_DOM_URL=ws://127.0.0.1
 ZONE_2_1_DOM_URL=ws://127.0.0.1
 ZONE_2_2_DOM_URL=ws://127.0.0.1
 
-# Sub websocket urls
+
+# Subordinate websocket urls
+
 PRIME_SUB_URLS=ws://127.0.0.1:8579,ws://127.0.0.1:8581,ws://127.0.0.1:8583
 REGION_0_SUB_URLS=ws://127.0.0.1:8611,ws://127.0.0.1:8643,ws://127.0.0.1:8675
 REGION_1_SUB_URLS=ws://127.0.0.1:8613,ws://127.0.0.1:8645,ws://127.0.0.1:8677
 REGION_2_SUB_URLS=ws://127.0.0.1:8615,ws://127.0.0.1:8647,ws://127.0.0.1:8679
 
-# Slices that are running
+
+# The SLICES variable determines which slices of the network the node will run.
+# The default configuration of 9 slices will run a "global node."
+# Selecting any subset of slices will run either a "multi-slice node" or a "slice node."
+
 SLICES="[0 0],[0 1],[0 2],[1 0],[1 1],[1 2],[2 0],[2 1],[2 2]"
 
-#Boolean Variable Definition
-#Enables or disables http porting
-#Enables or disables websocket porting
+
+# Boolean Variables
+
+# ENABLE_HTTP enables or disables http porting.
+# ENABLE_WS enables or disables websocket porting.
```

Comment:
s/porting/API/

Reply:
I have replaced "porting" with "API" in both lines.